contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,155 @@
+[certificates] Generated front-proxy CA certificate and key.
+[certificates] Generated front-proxy client certificate and key.
+[certificates] Valid certificates and keys now exist in "/etc/kubernetes/pki"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/admin.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/controller-manager.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/scheduler.conf"
+[apiclient] Created API client, waiting for the control plane to become ready
+[apiclient] All control plane components are healthy after 42.002391 seconds
+[token] Using token: 43a25d.420ff2e06336e4c1
+[apiconfig] Created RBAC rules
+[addons] Applied essential addon: kube-proxy
+[addons] Applied essential addon: kube-dns
+
+Your Kubernetes master has initialized successfully!
+
+To start using your cluster, you need to run (as a regular user):
+
+  mkdir -p $HOME/.kube
+  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+You should now deploy a pod network to the cluster.
+Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
+  http://kubernetes.io/docs/admin/addons/
+
+You can now join any number of machines by running the following on each node
+as root:
+
+  kubeadm join --token 43a25d.420ff2e06336e4c1 172.31.7.230:6443
+
+```
+
+## Step 3:
+Join the cluster from other hosts using kubeadm tool.
+
+```
+$sudo kubeadm join --token 43a25d.420ff2e06336e4c1 172.31.7.230:6443 --skip-preflight-checks
+[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.
+[preflight] Skipping pre-flight checks
+[discovery] Trying to connect to API Server "172.31.7.230:6443"
+[discovery] Created cluster-info discovery client, requesting info from "https://172.31.7.230:6443"
+[discovery] Cluster info signature and contents are valid, will use API Server "https://172.31.7.230:6443"
+[discovery] Successfully established connection with API Server "172.31.7.230:6443"
+[bootstrap] Detected server version: v1.7.6
+[bootstrap] The server supports the Certificates API (certificates.k8s.io/v1beta1)
+[csr] Created API client to obtain unique certificate for this node, generating keys and certificate signing request
+[csr] Received signed certificate from the API server, generating KubeConfig...
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+
+Node join complete:
+* Certificate signing request sent to master and response
+  received.
+* Kubelet informed of new secure connection details.
+
+Run 'kubectl get nodes' on the master to see this machine join.
+```
+
+At the end of Step 3 we will have a kubernetes cluster up and running and ready for deployment. Please follow the instructions [here](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) to deploy CNI network plugins and start a demo app.
```

Comment:
Can we put the CNI plugin into step 4. Although we are referencing other docs, but this step is very important, don't want user to miss it and thought the cluster is ready already.

Reply:
I would prefer all the post cluster bring up steps to be else where. The only reason is there are so many CNI plugins and we dont have to endorse any one. Secondly we let the user take it from here. For eg they just want to play with host network then they dont need cni plugin at all. So pointing to the doc might just make it work for us to keep this doc simple. Similar to what we do for ansible up top.

Reply:
@abhinandanpb I mean we could still reference the kubeadm doc, but we may want to put this into a step 4 just to address that there is still a step 4.

Reply:
Or else, it's really easy to ignore this small paragraph.

Reply:
ok got it . Will do